### PR TITLE
search: improve ES template for configuration

### PR DESCRIPTION
**Pull request by rerowep**
* Adds an setting template for elasticsearch with `number_of_shards`, `number_of_replicas` and `max_result_window`.
* Cleans all other elastic search mapping files.

Co-Authored-by:  Peter Weber <Peter.Weber@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

**Review by zannkukai — APPROVED** [review]

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit message proposition:

```
search: improve ES template for configuration

* Adds `number_of_shards`, `number_of_replicas` and `max_result_window`
  to record.json template.
* Cleans all other ElasticSearch mapping files.
```

File: rero_ils/modules/items/api/circulation.py [hunk 1]
```diff
@@ -166,7 +166,8 @@ def complete_action_missing_params(
 
         kwargs['item_pid'] = item_pid_to_object(item.pid)
 
-        kwargs['transaction_date'] = datetime.now(timezone.utc).isoformat()
+        # kwargs['transaction_date'] = datetime.now(timezone.utc).isoformat()
```
Comment (jma) [inline rero_ils/modules/items/api/circulation.py:169]: you can remove this

File: rero_ils/modules/items/api/circulation.py [hunk 2]
```diff
@@ -166,7 +166,8 @@ def complete_action_missing_params(
...
+        kwargs['transaction_date'] = datetime.utcnow().isoformat()
```
Comment (jma) [inline rero_ils/modules/items/api/circulation.py:170]: please mention this in the commit message

File: tests/ui/circulation/test_inhouse_cipo.py [hunk 3]
```diff
@@ -79,8 +79,9 @@ def test_less_than_one_day_checkout(
     # Loan date should be in UTC.
     loan_datetime = ciso8601.parse_datetime(loan_end_date)
     # Compare year, month and date
-    fail_msg = "Check timezone for Loan and Library. \
-It should be the same date, even if timezone changed."
+    fail_msg = "Check timezone for Loan and Library. " \
+               "It should be the same date, even if timezone changed."
     assert loan_datetime.year == next_open_day.year, fail_msg
     assert loan_datetime.month == next_open_day.month, fail_msg
+    # TODO: find a way for the test will work also after 23:00
```
Comment (jma) [inline tests/ui/circulation/test_inhouse_cipo.py:86]: please mention this in the commit message

**Review by jma — CHANGES_REQUESTED** [review]

**Review by AoNoOokami — APPROVED** [review]

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

**Review by jma — APPROVED** [review]

**Pull request merged**